frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -1,18 +1,136 @@
-import React from "react";
-import { GridContainer } from "@trussworks/react-uswds";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GridContainer, Icon, SiteAlert } from "@trussworks/react-uswds";
 
 import { AuthElement } from "../AuthElement";
 import { withCatchAndSuspense } from "../RSErrorBoundary";
+import { USLink } from "../USLink";
+import { showError } from "../AlertNotifications";
 import { MemberType } from "../../hooks/UseOktaMemberships";
+import { validateFileType, validateFileSize } from "../../utils/FileUtils";
+
+import ManagePublicKeyChooseSender from "./ManagePublicKeyChooseSender";
+import ManagePublicKeyUpload from "./ManagePublicKeyUpload";
+
+export const CONTENT_TYPE = "application/x-x509-ca-cert";
+export const FORMAT = "PEM";
+
+function ManagePublicKeySwitchDisplay() {
+    const navigate = useNavigate();
+    const [sender, setSender] = useState("");
+    const [fileContent, setFileContent] = useState("");
+    const [file, setFile] = useState<File | null>(null);
+    const [fileSubmitted, setFileSubmitted] = useState(false);
+
+    // TODO: mocked for now - make the call you need when sending the file
+    const { sendFile } = {
+        sendFile: (data: {
+            contentType?: string;
+            fileContent?: string;
+            file?: File | null;
+        }) => {
+            data = { file: null };
+            return data;
+        },
+    };
+
+    const onSenderSelect = (selectedSender: string) => {
```

Comment:
Also, I think it helps to differentiate the naming convention to know whether or not a function is defined within the current component or if it's coming down as a prop.  I tend to use `handle<object><action>` for functions defined within components and `on<object><action>` for names of props:

```tsx
// in ManagePublicKey

// defining in component: use handle*
function handleSenderSelect(selectedSender: string) {
    setSender(selectedSender);
}

// passing as a prop: use on*
onSenderSelect={handleSenderSelect}
```

This helps make it clearer when we're invoking callbacks passed from above as props too.  Like if there were _another_ callback we had to call:

```tsx
// if we want to trigger higher-level side-effects when the Sender is selected,
// we might want to pass in an onSenderSelect into ManagePublicKey

function ManagePublicKey({ onSenderSelect }) {
    function handleSenderSelect(selectedSender: string) {
        // manage local state
        setSender(selectedSender);
   
        // trigger the higher-level side-effects
        onSenderSelect(selectedSender);
    }
}

// easier differentiation if we need to pass this down as well
onSenderSelect={ handleSenderSelect }
```